app/uk/gov/hmrc/gform/validation/ComponentValidator.scala

```diff
@@ -388,7 +388,12 @@ object ComponentValidator {
       case tooShort if tooShort.length < 7 =>
         val vars: List[String] = 7.toString :: Nil
         validationFailure(fieldValue, genericErrorMinLength, Some(vars))
-      case Standard()   => validationSuccess
+      case Standard(_, s) =>
```

Comment:
should it be 
```
case Standard(_, s) if VatReferenceChecker.isValid(s) => validationSuccess
```
and the failure case will be handled by `case _ => ...` line 400

Reply:
Good catch 👍